base/fft/FFTW.jl

```diff
@@ -101,16 +101,16 @@ alignment_of(A::FakeArray) = Int32(0)
 # FFTW's api/import-wisdom-from-file.c file].
 
 function export_wisdom(fname::AbstractString)
-    f = ccall(:fopen, Ptr{Void}, (Cstring,Ptr{UInt8}), fname, "w")
+    f = ccall(:fopen, Ptr{Void}, (Cstring,Cstring), fname, "w")
```

Comment:
A cleaner solution: change these arguments to symbols. `convert` already skips null checking for them since they cannot contain NULLs.